### PR TITLE
simplify the way of getting message []byte format

### PR DESCRIPTION
**Pull request by gy95**
Signed-off-by: gy95 <guoyao17@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
`model.Message` has provided the function `GetContentData` to get `[]byte` format of message body,
So we can use function instead of implementing function itself.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/metamanager/client/configmap.go [hunk 1]
```diff
@@ -58,15 +58,9 @@ func (c *configMaps) Get(name string) (*api.ConfigMap, error) {
...
-	case []byte:
-		content = msg.GetContent().([]byte)
-	default:
-		content, err = json.Marshal(msg.Content)
-		if err != nil {
-			return nil, fmt.Errorf("marshal message to configmap failed, err: %v", err)
-		}
+	content, err := msg.GetContentData()
+	if err != nil {
+		return nil, fmt.Errorf("marshal message to configmap failed, err: %v", err)
```
Comment (Iceber) [inline edge/pkg/metamanager/client/configmap.go:63]: For `GetContentData` that return an error containing `marshal message content failed`.
https://github.com/kubeedge/kubeedge/blob/c51143389cdd3061a804b73be21b951c446bb68a/staging/src/github.com/kubeedge/beehive/pkg/core/model/message.go#L142-L152

I recommend avoiding including marshal-related information in an error

Comment (gy95) [reply edge/pkg/metamanager/client/configmap.go]: done, modify to `parse message to configmap failed`

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**